src/htr2hpc/train/run.py

```diff
@@ -239,8 +246,21 @@ def main():
         job_output = args.work_dir / f"segtrain_{job_id}.out"
         print(f"Job output should be in {job_output}")
 
-        # TODO: after run completes, check for results
+        # change back to original working directory
+        os.chdir(orig_working_dir)
+
+        # after run completes, check for results
         # - for segmentation, upload all models to eScriptorium as new models
+        upload_count = upload_models(
+            api,
+            output_modelfile.parent,
+            es_model_jobs[args.mode],
+            show_progress=args.show_progress,
+        )
+        # - does this behavior depend on job exit status?
+        # reasonable to assume any model files created should be uploaded¿
```

Comment:
for segmentation, yes any model files in the directory should be uploaded, even if the training process exits early for some reason. we will want to discuss best practices for handling transcription results when a `_best` model is not produced.